cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -51,9 +51,9 @@ function FilterableListControls( element ) {
     },
     email: function( field, currentStatus ) {
       var status = currentStatus || {};
-      var regex = [ "^[a-z0-9\u007F-\uffff!#$%&'*+\/=?^_`{|}~-]+(?:\.[a-z0-9",
-                    "\u007F-\uffff!#$%&'*+\/=?^_`{|}~-]+)*@(?:[a-z0-9]",
-                    "(?:[a-z0-9-]*[a-z0-9])?\.)+[a-z]{2,}$" ].join( '' );
+      var regex = [ '^[a-z0-9\u007F-\uffff!#$%&\'*+\/=?^_`{|}~-]+(?:\.[a-z0-9',
+                    '\u007F-\uffff!#$%&\'*+\/=?^_`{|}~-]+)*@(?:[a-z0-9]',
+                    '(?:[a-z0-9-]*[a-z0-9])?\.)+[a-z]{2,}$' ].join( '' );
```

Comment:
@sebworks @anselmbradford I'm going to abstract the validators to their own util file so that they're testable. Currently they're private making it difficult. See Jira issue#922